### ISSUE TITLE
Run virsort2 with multiple genomes 

### ISSUE THREAD

**Issue by whitnersyrena** [jiarong/VirSorter2#122]
Hi there, 

More of a general question, but I can't seem to locate this information. Is it possible to run virsorter2 on multiple genomes at the same time? I have 450 individual WGS fungal strains and wanted to screen them all individually. 

Thanks!

**Comment by jiarong** [comment]
Yes, VS2 treats each sequence independently. You can concatenate the genomes (fasta format) and feed it as input, or run each genome sequence separately in parallel, which will be much faster.

**Comment by matheus1617c** [comment]
Hello there, just use this command that it will run multiple sequences individually, but not at the same time.
Also, this command will create an individual directory for each of the sequences:

for i in *.fna; do virsorter run -w VirSorter2_results_$i -i $i --jobs 10 --min-length 1000 --keep-original-seq --exclude-lt2gene --rm-tmpdir all; done

Just add the parameters that you like and go for it and do not delete  this part 'rm-tmpdir'.
